linter.py

```diff
@@ -52,10 +36,10 @@ def find_gopaths(self):
             if p not in goroot and p not in gopath:
                 gopath.append(p)
 
-        if persist.debug_mode():
-            persist.printf("{}: {} {}".format(self.name,
-                                              os.path.basename(self.filename or '<unsaved>'),
-                                              "guessed GOPATH=" + os.pathsep.join(gopath)))
+        persist.debug("{}: {} {}".format(
```

Comment:
`persist.debug` is deprecated.

Define ```logger = logging.getLogger('SublimeLinter.plugin.golint')``` at the top. And the just use `logger.info(...)`

Reply:
Yeah I was wondering about that and couldn't find a good example. Thanks.